main/inc/lib/fileManage.lib.php

```diff
@@ -192,6 +192,9 @@ function move($source, $target, $forceMove = true, $moveContent = false)
 
         /* File case */
         if (is_file($source)) {
+            if (is_file($target . '/' . $file_name)) {
```

Comment:
I think we should also take the $forceMove param into account: $forceMove==true should imply we move the file even if the destination already exists. What do you think?